requirements.txt

```
@@ -1,6 +1,6 @@
 numpy>=1.17
 scipy>=1.4
-qiskit-terra>=0.19.0
+qiskit-terra>=0.20.0
```

This isn't actually required, it is only for the RB test (and the required bug fix PR hasn't been released in stable yet, it should be in 0.20.1)
```suggestion
qiskit-terra>=0.19.0
```